### PR TITLE
Remove chat option from on-demand

### PR DESCRIPTION
**Pull request by crichID**
## Overview
**TL;DR**
This PR removes the Chat option from on-demand. 

### Why

The chat option was always an experiment to see how many users would drop in and request help. After having the option available for ~ 1 year, we have found that a very small number of students use it. Most of these students also requested help in a GitHub issue or pull request, making chat redundant. 

This PR removes the chat option and attempts to reinforce the :+1: behavior of asking questions and requesting help in issues and pull requests.

### Next Steps
- [x] Get :eyes: from @hectorsector to make sure I removed everything that needed to be removed
- [x] :ship: it

/FYI @github/services-training for updates to comms

### CODE REVIEW

**Comment by crichID** [comment]
@hectorsector thank you for your quick review. I am going to go ahead and :ship: this one!
